src/js/tech/tech.js

```diff
@@ -313,6 +345,30 @@ class Tech extends Component {
     }));
   }
 
+
+  /**
+   * Initialize audiotrack listeners
+   *
+   * @method initAudioTrackListeners
+   */
+  initAudioTrackListeners() {
```

Comment:
should we keep initTextTrackListeners? since its public and we do not want to change the api.